### ISSUE TITLE
Can fail for regression if no meta-data found

### ISSUE THREAD

**Issue by mfeurer** [automl/auto-sklearn#110]
```
Process pynisher function call:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/home/feurerm/virtualenvs/ml2016/local/lib/python2.7/site-packages/pynisher/limit_function_call.py", line 83, in subprocess_func
    return_value = ((func(*args, **kwargs), 0))
  File "/home/feurerm/projects/automl_competition_2015/auto-sklearn/autosklearn/smbo.py", line 816, in run_smbo
    dataset_minimum = np.min(Y_meta[start_index:end_index, 0])
  File "/home/feurerm/virtualenvs/ml2016/local/lib/python2.7/site-packages/numpy/core/fromnumeric.py", line 2359, in amin
    out=out, keepdims=keepdims)
  File "/home/feurerm/virtualenvs/ml2016/local/lib/python2.7/site-packages/numpy/core/_methods.py", line 29, in _amin
    return umr_minimum(a, axis, None, out, keepdims)
ValueError: zero-size array to reduction operation minimum which has no identity
```

**Comment by dchansen** [comment]
Can confirm. In fact it seems that the regression module will not work at all if the "random_forest" estimator is excluded.

**Comment by mfeurer** [comment]
Original issue is solved by c304f3c198501d262b644c5b6812c0ae7420643b in the development branch. @dchansen I'm closing this now. If your issue persists, please open a new github issue.
